xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -110,8 +110,17 @@
       </ReturnValue>
       <Docs>
         <summary>Gets or sets the type of decompression method used by the handler for automatic decompression of the HTTP content response.</summary>
-        <value>The automatic decompression method used by the handler. The default value is <see cref="F:System.Net.DecompressionMethods.None" />.</value>
-        <remarks>To be added.</remarks>
+        <value>The automatic decompression method used by the handler.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks  
+For the .NET Framework 4.x <xref:System.Net.Http> binary in the Global Assembly Cache (GAC), the default value is <xref:System.Net.DecompressionMethods.None>. However, the .NET Core default is <xref:System.Net.DecompressionMethods.GZip> and <xref:System.Net.DecompressionMethods.Deflate>.
```

Comment:
Rather than "However...", I'd just make the second sentence parallel to the first. "For .NET Core..."